analyticscloud/importers/db.py

```diff
@@ -23,6 +23,9 @@
     sqltypes.REAL, sqltypes.SMALLINT
 )
 
+SQL_FLOAT_TYPES = (sqltypes.NUMERIC, sqltypes.Numeric, sqltypes.DECIMAL,
```

Comment:
We're using both in `SQL_NUMERIC_TYPES` as well. I'll do some quick testing to see which one actually gets returned from the introspection and update both instances. I know we've ran into some problems by not including enough of these classes in our other SQLAlchemy work, but we may have just been bulldozing over the problem.